code/__HELPERS/icons.dm

```diff
@@ -1342,5 +1342,5 @@ GLOBAL_LIST_EMPTY(transformation_animation_objects)
 	else
 		icon_states_cache[file][state] = FALSE
 		if(scream)
-			stack_trace("Icon Lookup for state: [state] in file [file] failed.")
+			THROW_ERROR("Icon Lookup for state: \"[state]\" in file \"[file]\" failed.")
```

Comment:
do you not want the stack trace here in case something higher up is sending a garbage icon state name? 

otherwise it's a lot harder to track down.

Frankly I don't see the value of this macro at all.

I'd rather use the disk space and have the stack trace than not have it and need it.